includes/admin/class-wc-admin-taxonomies.php

```diff
@@ -153,6 +153,24 @@ public function add_category_fields() {
 					jQuery( '.remove_image_button' ).hide();
 					return false;
 				});
+				
+				jQuery( document ).ajaxComplete( function( event, request, options ) {
+					if ( request && request.readyState === 4 && request.status === 200
```

Comment:
These conditions/checks should be "yoda" style conditions (https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#yoda-conditions).